content/en/docs/reference/command-line-tools-reference/cloud-controller-manager.md

```diff
@@ -29,7 +29,7 @@ cloud-controller-manager [flags]
       <td colspan="2">--add-dir-header</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">If true, adds the file directory to the header</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">If true, adds the file directory to the header of the log messages</td>
```

Comment:
Hello @alena1108 . Thanks for this PR.
All text updates for the component tools should be made directly on the component tool source code (--add-dir-header global flag), not on the individual document pages. The docs component tool pages are auto generated.

Reply:
@kbhawkey could you elaborate on `on the component tool source code (--add-dir-header global flag)` location? For this particular parameter, should the fix go to https://github.com/kubernetes/klog/blob/master/klog.go#L430?

Reply:
@alena1108 Yup. This should be fixed in the upstream source code.

Reply:
thank you for confirming @tengqm!